geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+semiexperimental and ab initio equilibrium structures of acetylene: Influence of relativistic effects and of the
+diagonal Born-Oppenheimer corrections. J Chem Phys. 134(6):064119/1-064119/8
+doi:10.1063/1.3553203 b.Tamassia F, Cané E, Fusina L, DiLonardo G (2016) The experimental equilibrium structure of acetylene. Phys
+Chem Chem Phys 18(3):1937-1944
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C  ≡ C 1.202958(7)
+C    H 1.06164(1)
+
+# Bond lengths: re (angstroms)
+
+C  ≡ C 1.202817(12)
+C    H 1.061689(23)
+
+NAME = C2H2O
+# Point Group: C2v
+# Shape: Trigonal planar
+# Bond lengths: r0 (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: a. Guarnieri A, Demaison J, Rudolph HD (2010) Structure of ketene – Revisited re (equilibrium) and rm (massdependent) structures. J Mol Struct 969(1-3):1-8 b. Atkinson SJ, Noble-Eddy R, Masters SL (2016) Gas-phase structures of ketene and acetic acid from acetic
+anhydride using very-high-temperature gas electron diffraction. J Phys Chem A 120 (12):2041-2048
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C = O     1.1628(101) 
+C = C     1.3145(106)
+C   H     1.0788(22)
+H   C = C 118.80(20)
+
+# Bond lengths: rs (angstroms)
+# Bond angles:  thetas (degrees)
+
+C = O     1.162(5)
+C = C     1.3136(56) 
+C   H     1.0795(17) 
+H   C = C 118.9(30)
+
+# Bond lengths: rm (angstroms)
+# Bond angles:  thetam (degrees)
+
+C = O     1.1617(4)
+C = C     1.3149(6) 
+C   H     1.1617(4)
+H   C = C 118.72(2)
+
+# Bond lengths: re^se (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+C = O     1.1607(6)
+C = C     1.3122(6)
+C   H     1.0763(1)
+H   C = C 119.115(11)
+
+# Bond lengths: rh1 (angstroms)
+# Bond angles:  thetah1 (degrees)
+
+C = O     1.154(3)
+C = C     1.323(2)
+C   H     1.085(1)
+H   C = C 119.4(5)
+H   C   H 121.3(10)
+
+# C = O 1.154(3), C = C 1.323(2), C H 1.085(1) and H C = C 119.4(5) constrained to the value from MP2/6-311++G** calculation. 
+# H C H 121.3(10) dependent parameter.
+
+NAME = C2H3As 2009
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Demaison J, Møllendal H., Guillemin JC (2009) Equilibrium CAs and CSb bond lengths. J Mol Struct 930(1-
+3):21-25
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C ≡ As  1.6585(2)
+C   C   1.4634(2)
+C   H   1.0910(3)
+H   C C 110.575(10)
+
+NAME = C2H5N 
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu X, Xu Y (2011) Infrared and microwave spectra of the acetylene-ammonia and carbonyl sulfide-ammonia
+complexes: a comparative study of a weak C-H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+H...N    2.3980(7)
+NH3 HCCH 4.0594(6)
+α        23.29(2)
```

Comment:
I commented about alpha earlier, but I think that this is essentially the CHN bond angle (in this case, and something similar in the other case), although this isn't clear from the book. We might have to look at the paper to confirm what this angle is.